### PR TITLE
Update base32 address

### PR DESCRIPTION
**Pull request by Pana**
Also remove the js-conflux-sdk part

---
This change is [Reviewable](https://reviewable.io/reviews/conflux-chain/conflux-doc/59)

### CODE REVIEW

File: docs/json-rpc.md [hunk 1]
```diff
@@ -20,10 +20,6 @@ JSON is a lightweight data-interchange format. It can represent numbers, strings
 
 JSON-RPC is a stateless, light-weight remote procedure call (RPC) protocol. Primarily this specification defines several data structures and the rules around their processing. It is transport-agnostic in that the concepts can be used within the same process, over sockets, over HTTP, or in many various message passing environments. It uses JSON (RFC 4627) as its data format.
 
-### JavaScript API
-
-We also provide a JavaScript library [js-conflux-sdk](https://github.com/Conflux-Chain/js-conflux-sdk) for you to interact with a Conflux node from a JavaScript application. This  SDK offers a convenient interface on top of the RPC methods.
-
```
Comment (Thegaram) [inline docs/json-rpc.md:26]: I think it would be best to cross-reference the SDK section here once it's ready. For now we can remove it.

**Review by Thegaram — APPROVED** [review]

File: docs/json-rpc.md (hunk 1 above)
Comment (Pana) [reply docs/json-rpc.md:26]: I think the fullnode RPC is  a root, all the SDK's documentation will have reference to here, but seems fullnode's RPC does not need to link to SDK's API

Comment (Thegaram) [reply docs/json-rpc.md:26]: Why not have references both ways? If someone stumbles upon the RPC docs, they might be pleasantly surprised to find out we offer so many SDKs. It's up to you though, you're free to merge this PR.

Comment (Pana) [reply docs/json-rpc.md:26]: If add SDK's link here, there will be four SDK listed here

**Pull request merged**